### DISCUSSION TITLE
Bring the old home page the new home page is horrible

### DISCUSSION THREAD

**Discussion by sanidhyy** [community/community#66559] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The new home feed is horrible and full of OTHER stuff. I can't focus on my activities😢.

The old feed is still here https://github.com/dashboard-feed - Please GitHub do the right thing and fix home feed 🙌
This topic was also covered by HackerNews https://news.ycombinator.com/item?id=37413126

**Comment by AmberSolstice** [comment]
yess

**Comment by Vincentqyw** [comment]
+10000
